pom.xml

```diff
@@ -518,6 +518,11 @@
                 <artifactId>opentelemetry-extension-trace-propagators</artifactId>
                 <version>${otel.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.opentelemetry</groupId>
```

Comment:
Since we are adding a new dependency, shall we run the dependency scanner for enforcer ?

Reply:
No dependency related vulnerabilities detected in the dependency scanner.